tasks/apt/install.yml

```diff
@@ -45,13 +39,13 @@
     group: root
     mode: 0644
 
-- name: Install OpenJDK 7
+- name: Install OpenJDK 8
   apt:
-    name: openjdk-7-jdk
+    name: openjdk-8-jdk
```

Comment:
Sounds good; I like the idea of a `jenkins_apt_packages` variable, since that would also give users the ability to easily install apt packages on the Jenkins server while deploying with this role. Fixup is coming soon.